.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+require:
```

Comment:
We switched from RuboCop to [standard](https://github.com/testdouble/standard) recently. Given the no-configuration philosophy of standard, I would prefer to leave this out. I think the gemspec is unlikely to change frequently anyway, so leaving off the rule seems OK.